pandas/core/frame.py

```diff
@@ -2185,7 +2185,7 @@ def __getitem__(self, key):
         try:
             if key in self.columns and not is_mi_columns:
                 return self._getitem_column(key)
-        except:
+        except Exception:
```

Comment:
either leave these  (IOW the linter is not actually failing these on CI), or catch a specific exception